src/app/modules/dashboard/modules/day-plan/components/day-plan-stats/day-plan-stats.component.scss

```diff
@@ -0,0 +1,87 @@
+@import "colors.scss";
+@import 'layout.scss';
+
+.day-plan-summary-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  border-top-right-radius: 3rem;
+  border-top-left-radius: 3rem;
+  box-shadow: transparentize(#ca2043, 0.5) 0px -2px 20px 0px;
+  padding: 0.25rem 0.5rem;
+  background-color: #f4d2b5;
+  position: sticky;
+  width: 100%;
+  bottom: 0;
+}
+.day-plan-summary-wrapper-hidden {
+  transform: translateY(67%);
+}
+.summary-element {
+  display: flex;
+  flex-direction: column;
```

Comment:
to bym prędzej na row zrobił

Reply:
to jest wrapper dla tego nad barem i dla samego bara. jak dam na row to sie ustawia obok siebie